### DISCUSSION TITLE
Docs only: clarify Leaky Vessels CVEs impacted versions

### DISCUSSION THREAD

**Discussion by stone-z** [moby/moby#47614] [Q&A]
### Description

The [v24.0.9 release notes](https://github.com/moby/moby/releases/tag/v24.0.9) state:

> Important ⚠️
    Note that this release of Docker Engine doesn't include fixes for the following known vulnerabilities in BuildKit:
        [CVE-2024-23651](https://scout.docker.com/v/CVE-2024-23651)
        [CVE-2024-23652](https://scout.docker.com/v/CVE-2024-23652)
        [CVE-2024-23653](https://scout.docker.com/v/CVE-2024-23653)
        [CVE-2024-23650](https://scout.docker.com/v/CVE-2024-23650)
    To address these vulnerabilities, upgrade to [Docker Engine v25.0.2](https://github.com/moby/moby/blob/v24.0.9/25.0.md#2502).

> For more information about the security issues addressed in this release, and the unaddressed vulnerabilities in BuildKit, refer to the [blog post](https://www.docker.com/blog/docker-security-advisory-multiple-vulnerabilities-in-runc-buildkit-and-moby/).

However, the linked Docker blog post lists moby v24.0.9 as a patched version.
![image](https://github.com/moby/moby/assets/10408416/5ef48572-a160-46f6-b082-397a36a40cbb)

Which is correct? These sources should be aligned.

### Reproduce

1. Read the [v24.0.9 release notes](https://github.com/moby/moby/releases/tag/v24.0.9)
2. Read the [blog post](https://www.docker.com/blog/docker-security-advisory-multiple-vulnerabilities-in-runc-buildkit-and-moby/)

### docker version

```bash
v24.0.9
```

### docker info

```bash
N/A
```

**Comment by thaJeztah** [comment]
I asked the team maintaining the website to make this more clear, and they added a footnote to outline the CVEs not fixed in 24.0.9;

<img width="883" alt="Screenshot 2024-02-14 at 16 34 45" src="https://github.com/moby/moby/assets/1804568/7cd39cc3-e565-49c9-9104-ee7387d80ecc">


I'll close this ticket, as the blog-post is maintained by Docker, not the project maintainers for the Moby code, but feel free to continue the conversation.

**Comment by crossterra-agent** [comment]
- **Status**: DONE
- **Role**: orchestrator
- **Step**: 
- **Link**: https://github.com/CrossTerra/monorepo/pull/2512#issuecomment-4361833713
- **Summary**: Requested completion review on PR #2512 — developer confirmed all steps complete with CI passing.
